qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/30_camel_case_dateformat.yml

```
@@ -0,0 +1,79 @@
+---
+"Create index with camel case on date format (allowed with warning prior to 2.12.0)":
+  - skip:
+      version: "2.12.0 - "
+      reason:  "at version 2.12.0, camel case date format is not allowed"
+      features: "warnings"
+  - do:
+      warnings:
+        - "Camel case format name strictDateOptionalTime is deprecated and will be removed in a future version. Use snake case name strict_date_optional_time instead."
+      indices.create:
+        index: camel_case_date_format
+        body:
+          settings:
+            index:
+              number_of_replicas: 2
+          mappings:
+            "properties":
+              "date_field":
+                "type": "date"
+                "format": "strictDateOptionalTime"
+
+
+  - do:
+      bulk:
+        refresh: true
+        body:
+          - '{"index": {"_index": "camel_case_date_format"}}'
+          - '{"date_field": "2023-02-01T00:00+01:00"}'
+
+  - do:
+      search:
+        rest_total_hits_as_int: true
+        index: camel_case_date_format
+        body:
+          query:
+            range:
+              date_field:
+                gte: "2023-01-01T00:00+01:00"
+                lte: "2023-03-01T00:00+01:00"
+  - match: { hits.total: 1 }
+
+---
+"Create index with camel case date format (when bwc version is > 8.0.0)":
```

Not sure where 8.0.0 came from?

```suggestion
"Create index with camel case date format (when bwc version is > 1.0.0)":
```